frameworks/Python/django/hello/world/views.py

```diff
@@ -26,60 +29,48 @@ def plaintext(request):
 
 
 def json(request):
-    response = {
-        "message": "Hello, World!"
-    }
-    return HttpResponse(uj_dumps(response), content_type="application/json")
+    return HttpResponse(
+            uj_dumps({"message": "Hello, World!"}),
+            content_type="application/json"
+        )
 
 
 def db(request):
-    r = random.randint(1, 10000)
-    world = uj_dumps({'id' : r, 'randomNumber' : World.objects.get(id=r).randomnumber})
+    r = _random_int()
+    world = uj_dumps({
+        'id' : r,
+        'randomNumber' : World.objects.only("randomnumber").get(id=r).randomnumber
```

Comment:
I realize that you're basically copying existing code, but it looks like this framework was in violation of the rules to begin with. [Rule number one](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#requirements-1)

If I understand this correctly, what's happening here is equivalent to
```
SELECT randomNumber from World where id=...
```

Reply:
@nbrady-techempower updated